src/test/resources/application.yml

```diff
@@ -0,0 +1,41 @@
+spring:
```

Comment:
I think we should be able to avoid copypasting the whole thing. Maybe with profiles ? Maybe src/test/resources/application-default.properties (kind of a hack)?